test/TestMain.hs

```diff
@@ -0,0 +1,11 @@
+
+module TestMain(runAllTests) where
+
+import Test(runTestSuites)
+import qualified TestLexer
+
+runAllTests :: IO ()
+runAllTests = runTestSuites [
```

Comment:
Would be great update the readme to say that when we want add new test cases, we should call that test here, right?